### DISCUSSION TITLE
Deployment to Azure-Playfab

### DISCUSSION THREAD

**Discussion by deepx** [microsoft/aspire#4140] [Ideas]
I am working with Aspire and Unrealengine to build up a MMORPG, and  bring successfully Aspire and the Infrastructure of UE5 together. So maybe it is a great wayto deploy to Playfab directly. What do you think about that?

![image](https://github.com/dotnet/aspire/assets/3179474/d60e4c03-c1b2-4601-a080-1c8f8ac59bcc)
![image](https://github.com/dotnet/aspire/assets/3179474/de25a0c6-0b4d-4148-943a-9b95913f1f3e)

greets DeepX

**Comment by davidfowl** [comment]
Sounds great, but it's not a primary scenario so it's not something the team will prioritize. It would be cool to see if you can build on top of what is there to enable it (or at least find the gaps).

**Comment by andhesky** [comment]
@deepx, 

PlayFab Multiplayer Servers (MPS) is generally best suited for session-based gameplay. I am making an assumption your servers are long lived if it is an MMORPG so it might not be the best fit in your case.

The overall idea of being able to push to MPS or configure other parts of PlayFab from Aspire is still interesting, but we will need to consider the right scenario to make it valuable.
